### PR TITLE
Format fixes and other updates

### LINKED ISSUE #20: New logo goes into navigation bar on mobile
The red 'SPACE STATION 3D' moves up into the navigation bar on mobile or skinny desktop window. It should remain on the top of the site instead.

[image](https://i.imgur.com/Y9kvXUM.png)

Solving #17 would also solve this.

### PR DESCRIPTION
**Pull request by cosmiccoincidence**
- Removes icons from navbar
- Tweaks margins
- Centers text on the Artwork page

This should fix #20, and partially solves #17, but to fully fix it would require removal of the square in the top right of the screen (in the navbar) seen on mobile.

---

Now also...

- Changes header text to red
- Changes background color on light theme
- Adds animation kit asset
- Updates contribution page with additional links

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
-            <source srcset="{{ site.baseurl}}/assets/img/logo.webp" type="image/webp">
-            <source srcset="{{ site.baseurl}}/assets/img/logo.png" type="image/png"> 
-            <img alt="" src="{{ site.baseurl}}/assets/img/logo.png">
-        </picture>
-        <picture class="logo" >
-            <source srcset="{{ site.baseurl}}/assets/img/SPACESTATION3D.webp" type="image/webp">
-            <source srcset="{{ site.baseurl}}/assets/img/SPACESTATION3D.png" type="image/png"> 
-            <img alt="" src="{{ site.baseurl}}/assets/img/SPACESTATION3D.png">
-        </picture>
+<center>
```
Comment (TimFinucane) [inline _includes/header.html]: This tag isn't recommended in HTML5. Should instead add a style (text-align: center) to the "header" class in the css.

File: artwork.md [hunk 2]
```diff
@@ -7,10 +7,12 @@ permalink: /artwork/
 This Page is for displaying SS3D-related artworks such as wallpapers, animations and music created by artists of the community.
 
 # Brukas
+{: class="centred" }
```
Comment (TimFinucane) [inline artwork.md]: Might be better to make all headers centred in the CSS, if that's what we actually want?

**Comment by TimFinucane** [comment]
Final thing is that the header is slightly to the right on small (~300px) screens. this is due to the padding of the image in the header pushing it to the right.

File: _includes/header.html (hunk 1 above)
Comment (cosmiccoincidence) [reply _includes/header.html]: I didn't even need this at all... -_-

File: artwork.md (hunk 2 above)
Comment (cosmiccoincidence) [reply artwork.md]: I tried this originally but couldn't figure out how to make that happen for only the artwork page in the short time I looked at it. So this was my short term solution, but if you can point me in the right direction on how to make it for artwork page only, I can try to do so... it seems I might need a new layout.html?

**Comment by cosmiccoincidence** [comment]
> Final thing is that the header is slightly to the right on small (~300px) screens. this is due to the padding of the image in the header pushing it to the right.

This should be fixed more properly with additional breakpoints as opposed to the 2 we currently have I believe, which is something E3 brought to my attention. For now I fixed the issue by decreasing the img size.

**Comment by cosmiccoincidence** [comment]
Added a few things that could have been done in a different PR but oh well... such as...

- changing background color on light theme (Dyhr's suggestion)
- adding Probably Not's animation kit to the contribute page

File: artwork.md (hunk 2 above)
Comment (TimFinucane) [reply artwork.md]: Yeah that would be correct unfortunately. I wouldn't object to you doing it this way for now as long as we try to fix it in the future

Comment (cosmiccoincidence) [reply artwork.md]: Yeah, I would also prefer it that way as well but it's currently beyond my knowledge and time.

**Pull request merged**